landlab/components/stream_power/cfuncs.pyx

```diff
@@ -4,6 +4,9 @@ cimport cython
 
 #from libc.math cimport fabs
 
+# as best as KRB can tell, the function _brentq is in c.
+# not sure what, if anything I need to do to 
+from scipy.optimize._zeros import _brentq as brentq
 
```

Comment:
IIRC I wrote out the Newton method longhand in my original version, for this exact reason. (Could be remembering this wrong, but I definitely also hit this issue and concluded import was not ideal, for this reason).